File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String input = sc.nextLine();

        if (input.contains("c=")) {
            input = input.replace("c=", "@");
        }
        if (input.contains("c-")) {
            input = input.replace("c-", "@");
        }
        if (input.contains("dz=")) {
            input = input.replace("dz=", "@");
        }
        if (input.contains("d-")) {
            input = input.replace("d-", "@");
        }
        if (input.contains("lj")) {
            input = input.replace("lj", "@");
        }
        if (input.contains("nj")) {
            input = input.replace("nj", "@");
        }
        if (input.contains("s=")) {
            input = input.replace("s=", "@");
        }
        if (input.contains("z=")) {
            input = input.replace("z=", "@");
        }
        System.out.println(input.length());
    }
}

File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.py
str = input()
# if 'c=' in str:
#     str = str.replace('c=','@')
# if 'c-' in str:
#     str = str.replace('c-','@')
# if 'dz=' in str:
#     str = str.replace('dz=','@')
# if 'd-' in str:
#     str = str.replace('d-','@')
# if 'lj' in str:
#     str = str.replace('lj','@')
# if 'nj' in str:
#     str = str.replace('nj','@')
# if 's=' in str:
#     str = str.replace('s=','@')
# if 'z=' in str:
#     str = str.replace('z=','@')
# print(len(str))

alpha = ['z=', 's=', 'nj', 'lj', 'd-', 'dz=', 'c-', 'c=']
for i in alpha[::-1]:
    str = str.replace(i, '@')

print(len(str))